projects/storefrontapp-e2e-cypress/cypress/integration/regression/customer-ticketing/view-ticket-details.e2e-spec.ts

```diff
@@ -17,6 +17,21 @@ describe('ticketing', () => {
         customerTicketing.clickFirstTicketFromTicketListing();
         customerTicketing.verifyTicketDetailsPageVisit();
       });
+      it('should be able to visit ticket details page for an existing ticket via url', () => {
+        customerTicketing.loginRegisteredUser();
+        customerTicketing.clickMyAccountMenuOption();
+        customerTicketing.clickCustomerSupportMenuOption();
+        customerTicketing.verifyTicketListingPageVisit();
+        customerTicketing.visitTicketDetailsPageForFirstTicket();
+      });
+      it('should throw 404 error when trying to visit ticket details page for a non-existing ticket id via url', () => {
+        customerTicketing.loginRegisteredUser();
+        customerTicketing.clickMyAccountMenuOption();
+        customerTicketing.clickCustomerSupportMenuOption();
+        customerTicketing.verifyTicketListingPageVisit();
```

Comment:
I guess we don't need these lines since we are navigating by URL to ticket details page